packages/components/forma-36-react-timepicker/src/TimeZonePicker.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ChangeEvent } from 'react';
+import { Select, Option } from '@contentful/forma-36-react-components';
+import { defaultZoneOffset, zoneOffsets } from './utils/zoneOffsets';
+
+export type TimeZonePickerProps = {
+  disabled: boolean;
+  onChange: (value: string) => void;
+  value?: string;
+  className?: string;
+};
+
+const TimeZonePicker: React.FC<TimeZonePickerProps> = ({
+  disabled,
+  onChange,
+  className,
+  value = defaultZoneOffset,
+}) => {
+  return (
+    <Select
+      className={className}
+      aria-label="Select timezone"
+      testId="timezone-input"
+      value={value}
+      width="medium"
+      isDisabled={disabled}
+      onChange={(e: ChangeEvent<HTMLSelectElement>) => {
+        onChange(e.currentTarget.value);
+      }}
+    >
+      {zoneOffsets.map(offset => (
```

Comment:
this might work for current times but not for future looking dates since you'll not be able to respect daylight saving time when scheduling something. E.G: Berlin and Cairo are on the same offset in summer but not in winter. If I'm not able to select the the TZ database name its not possible to schedule the correct future time for a certain timezone.

Reply:
Do you think it would be better to use some sort of framework for displaying timezones with daylight savings times? should this be a component at all?

Reply:
I think we need to bring this up in the frontend/backend chapter how we want to store dates in the future. Using offsets will definitely result in wrong future looking dates thats why we chose to use the TZ Name for scheduled publishing. For now and to migrate it it should be fine to continue with the offsets.